docs/okeps/okep-5252-multiple-primary-udns.md

```diff
@@ -0,0 +1,208 @@
+# OKEP-5252: Multiple Primary UDNs in a namespace
+
+* Issue: [#5252](https://github.com/ovn-org/ovn-kubernetes/issues/5252)
+
+## Problem Statement
+
+When a primary UDN runs out of address space, the only option today is to delete the
```

Comment:
This is one of the use cases we have covered for connecting UDNs https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5246 OKEP as well